src/test/scala/de/unruh/isabelle/pure/TheoryHeader.scala

```diff
@@ -0,0 +1,19 @@
+package de.unruh.isabelle.pure
+
+import de.unruh.isabelle.control.IsabelleTest.isabelle
+import org.scalatest.funsuite.AnyFunSuite
+
+
+class TheoryHeaderTest extends AnyFunSuite {
+  test ("thy_header simple") {
+    val header = TheoryHeader.parse("theory Foo imports Bar Baz begin")
+    assert(header.name == "Foo")
+    assert(header.imports == List("Bar", "Baz"))
+  }
+  test ("thy_header hard") {
+    val t = "theory \"Foo-a_b\" imports \"../Bar-a_b\" Baz.Baz (* comment *) begin whatever"
+    val header = TheoryHeader.parse(t)
```

Comment:
I notice that `parse` ignores additional text after the theory header. I would call this unexpected. (If I parse an int, for example, I wouldn't expect "123hello" to successfully parse.) Can you document that in the docstring of `parse`?

Reply:
Ok.